bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat

```diff
@@ -92,20 +92,20 @@
 #*********************************************************************** 
 # Reweight variables for scale dependence and PDF uncertainty          *
 #***********************************************************************
- 1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
- 1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+ 1.0 = rw_rscale ! muR factors to be included by reweighting
+ 1.0 = rw_fscale ! muF factors to be included by reweighting
  True = reweight_scale ! Reweight to get scale variation using the 
```

Comment:
Annoyingly 1 of the 2 between reweight_PDF and reweight_scale have to be set to true for the weights to be stored. I asked Olivier about this and it seemed intentional